web/src/components/TempAlert.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { Alert, Button, CloseIcon } from '@cmsgov/design-system';
+
+import { getTempAlerts } from '../redux/reducers/alerts';
+
+const TempAlert = ({ alerts }) => {
+  const [tempAlertState, setTempAlertState] = useState(alerts);
+
+  if (!alerts) {
+    return null;
+  }
+
+  const removeMsg = i => {
```

Comment:
You should be managing this state in redux. Create a remove alert function in the tempAlert actions. Then use this logic to remove the alert from the messages array in redux.